templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -180,14 +180,15 @@ contents:
         extra_brex_args+="ipv6.addr-gen-mode ${ipv6_addr_gen_mode} "
       fi
 
-      # create bridge; use NM's ethernet device default route metric (100)
+      # create bridge; use a value lower than NM's ethernet device default route metric
+      # (we pick 49 to be lower than anything that NM chooses by default)
       if ! nmcli connection show "$bridge_name" &> /dev/null; then
         nmcli c add type ovs-bridge \
             con-name "$bridge_name" \
             conn.interface "$bridge_name" \
             802-3-ethernet.mtu ${iface_mtu} \
-            ipv4.route-metric 100 \
-            ipv6.route-metric 100 \
+            ipv4.route-metric 49 \
+            ipv6.route-metric 49 \
```

Comment:
could we use a variable?   not sure why we started with magic numbers.

Reply:
Done, thanks for the good suggestion. Should have been obvious :-)